### DISCUSSION TITLE
How to set solid background color instead of wallpaper? (Newbie)

### DISCUSSION THREAD

**Discussion by Dev-Sharbel** [mylinuxforwork/dotfiles#342] [Q&A]
Hello, I'd like to remove wallpapers and set a solid black color background.

This is my first time using hyprland so a lot of things are confusing.

I tried setting some `disable_hyprland_logo` and `background_color ` in .config/hypr/conf/custom.conf but nothing happened

**Comment by Veicm** [comment]
I'm also not very professionell but I would just Open a Browser and search "black" and select a random Image and Save in `~/wallpaper/` then open the pre installed Application waypaper and select the black Image. May you also have to turn off pywal with: ```sudo Pacman -Rns python-pywal```

But you should first try out with pywal still instaled (pywal Changes the Color of For example the terminal Text based on your wallpaper Color so maybe the Text will be unreadable).
Only uninstal it if you can't read the Text anymore.
